app/jobs/scraper/oregon_job.rb

```diff
@@ -38,7 +38,8 @@ def perform
             organization: organization
           ).first_or_initialize(
             notes: "From Scraper::OregonJob",
-            public_document: true
+            public_document: true,
+            source_url: apprenticeship_url
```

Comment:
I'm wondering if this should be `apprenticeship_url + "apprenticeship-opportunities.aspx"` ?

Reply:
That makes sense. I'll add that in